subworkflows/local/analyse_transcripts.nf

```diff
@@ -74,19 +75,22 @@ workflow ANALYSE_TRANSCRIPTS {
             drop_padjcutoff_as
         )
 
-        ch_out_drop_ae_rds    = DROP_CONFIG_RUN_AE.out.drop_ae_rds    ? DROP_CONFIG_RUN_AE.out.drop_ae_rds.collect()
-                                                                        : Channel.empty()
-        ch_out_drop_gene_name = DROP_CONFIG_RUN_AE.out.drop_gene_name ? DROP_CONFIG_RUN_AE.out.drop_gene_name.collect()
-                                                                        : Channel.empty()
-        ch_out_drop_as_tsv    = DROP_CONFIG_RUN_AS.out.drop_as_tsv    ? DROP_CONFIG_RUN_AS.out.drop_as_tsv.collect()
-                                                                        : Channel.empty()
+        ch_out_drop_ae_rds       = DROP_CONFIG_RUN_AE.out.drop_ae_rds       ? DROP_CONFIG_RUN_AE.out.drop_ae_rds.collect()
+                                                                            : Channel.empty()
```

Comment:
For consistency, I think its better to keek ´Channel.empty()´ because it is what has been used in most instances, like at workflows/tomte.nf. If you want, we can also discuss ^^